ArticleTemplates/assets/js/bootstraps/liveblog.js

```diff
@@ -312,7 +317,7 @@ define([
             window.liveblogTime();
             window.addEventListener('scroll', util.debounce(updateBlocksOnScroll, 100, true));
             liveMore();
-            
+
```

Comment:
We'll still need to call creativeInjector.trackLiveBlogEpic() from the ready function incase any are in the page from initialisation, calling it after liveMore() should be fine.